docs/sources/v0.53.x/javascript-api/k6-experimental/fs/file/read.md

```diff
@@ -39,6 +39,8 @@ let file;
 export default async function () {
   const buffer = new Uint8Array(128);
 
+  await file.seek(0, SeekMode.Start);
+
```

Comment:
In this example there is a seek at the end. I guess it is better if it is in the begging though ... in case of exceptions.

Reply:
Updated a bit

Reply:
I meant that in the end of the function there is a seek as well.

Reply:
Yes, but how it was before my changes, probably we can remove it if the same function is used at the start

Reply:
If it make sense for you, I will update it in such way

Reply:
> Yes, but how it was before my changes, probably we can remove it if the same function is used at the start

That was what I meant with my original comment - we don't need it twice, and it is better to be at the beginning

Reply:
@mstoykov Updated, please take a look :)